Constitution.md

```diff
@@ -145,43 +141,65 @@ If there is more then 1 president, they exist as a single voting member on counc
 3. Monitor maintenance of the CSSA Office, CSSA Lounge, and purchase pertinent supplies when needed.
 4. Ensure appropriate reservation, organization, and signage for events that take place in the CSSA Lounge.
 5. Ensure that the CSSA lounge is a safe and inclusive space.
-6. Lead the CSSA lounge supervisors and work with council in appointing them.
-7. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
+6. Work with the Director of Finances and the Finance Committee to record lounge transactions.
+7. Lead the Lounge Committee, call committee meetings and delegate tasks to committee members.
+8. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
 
 ### Director of Student Affairs
 
 1. Act as a liaison for the students towards the Department of Computer Science, Faculty of Science and UMSU.
 2. Act as a liaison between students and Council.
-3. Stay up to date on departmental and Senate meetings pertaining to the Computer Science program.
-4. Lead relevant campaigns on issues affecting the Computer Science Student Community.
-5. Work with the Director of Advocacy to promote opportunities of initiative/programs/organizations pertaining to where the Computer Science community is underrepresented.
-6. Ensure that the CSSA lounge is a safe and inclusive space.
-7. Lead the Outreach Committee with up to 3 committee members.
-8. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
+3. Stay up to date on Faculty and Senate meetings pertaining to the Computer Science program.
+4. Attend Student Executive Assembly (SEA) Meetings and Department of Computer Science Meetings on behalf of the CSSA, or put fair effort into acquiring a proxy from Council.
+5. Summarize the relevant contents of external meetings, including but not limited to those listed above, to Council.
```

Comment:
nit: can we just mention the meetings by name or refer directly to points 3 and 4.

Reply:
I assume to cover changes and additional ones that come up, eg the new science one

Reply:
Yeah my thought was that there may be some that aren't covered in points 3 and 4, I could make it less wordy by saying "external meetings" and cutting it off there